src/pages/docs/deployments/custom-scripts/run-a-script-step.md

```diff
@@ -147,6 +147,17 @@ If the package reference was _not_ configured to be extracted, then the un-extra
 
 These locations were designed to be convenient for use from custom scripts, as the relative path can be predicted, e.g. `./Acme` or `./Acme.zip`.  If the absolute path is required the variables above may be used.
 
+#### Docker image package variables
+In the scenario where your package reference is a Docker image some additional variables will be contributed. These variables are (assuming a package-reference named `Acme`):
+
+| Variable name and description                                                                                                                                | Example                      |
```

Comment:
This is missing `Octopus.Action.Package[Acme].Image.Label[Foo]` since the functionality is broken due to https://github.com/OctopusDeploy/Issues/issues/9163 and https://github.com/OctopusDeploy/Issues/issues/9164